flexmeasures/api/v2_0/tests/utils.py

```diff
@@ -144,10 +145,13 @@ def verify_sensor_data_in_db(
 
 
 def message_for_post_prognosis(fm_scheme: str = "fm1"):
+    """
+    Posting prognosis for a wind mill's production.
+    """
     message = {
         "type": "PostPrognosisRequest",
         "connection": f"ea1.2018-06.localhost:{fm_scheme}.2",
-        "values": [300, 300, 300, 0, 0, 300],
+        "values": [-300, -300, -300, 0, 0, -300],
```

Comment:
This seems like a consequence of changing ids, too. I remember writing this test prognosis for a Charge Point, not for a wind turbine.